src/tests/Button.test.js

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+
+// import sinon from 'sinon';
+
 import Enzyme, { shallow } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import Button from '../components/Button/Button';
+import { TokenBucket } from 'limiter';
```

Comment:
Where did you get limiter?

Reply:
🤔 I thought it was there, it came with the project.
If is not the case, maybe is a VSCode stuff. My VSCode sometime add, in a random fashion, 'import' statements at his own discretion.